src/github/createPRViewProvider.ts

```diff
@@ -214,20 +219,21 @@ export class CreatePullRequestViewProvider extends WebviewBase implements vscode
 
 		const defaultBranch = await githubRepository.getDefaultBranch();
 		const newBranches = await this._folderRepositoryManager.listBranches(owner, repositoryName);
-		this._onDidChangeSelectedRemote.fire({ owner, repositoryName });
+		this._onDidChangeBaseRemote.fire({ owner, repositoryName });
 		return this._replyMessage(message, { branches: newBranches, defaultBranch });
 	}
 
 	private async create(message: IRequestMessage<OctokitCommon.PullsCreateParams>): Promise<void> {
 		try {
-			if (!this._folderRepositoryManager.repository.state.HEAD!.upstream) {
+			// TODO@eamodio Why do we assume this is a detached head? if the upstream is missing isn't it just unpublished?
```

Comment:
Yeah this doesn't make sense, this check should just be completely removed. `createPullRequest` will check if there is an upstream and offer to publish if not. Thanks for catching!